src/options.ts

```diff
@@ -25,6 +25,7 @@ export enum ServiceOption {
   GRPC = 'grpc-js',
   GENERIC = 'generic-definitions',
   DEFAULT = 'default',
+  GENERIC_AND_SERVICES = 'generic-definitions-and-services',
```

Comment:
Hm, kinda think, let's maybe not add a new definition and instead change `options.outServices` to `ServiceOption[]`?

I was thinking like `outputServices=generic,default`, but `,` is already taken as a separator.

We could list options twice, like `outputServices=generic-definitions,outputServices=default`... I just made a fix to `snakeToCamel` that provides a precedent for how to support this "nudged to an array" pattern for options:

https://github.com/stephenh/ts-proto/pull/513

Does that seem okay?

Reply:
Great!